File: Lab 3/PersonalHome.java
public class PersonalHome extends Accommodation{
    private int numberOfFloors;

    public PersonalHome(int numberOfFloors, int numberOfRooms, double price, double area, double latitude, double height,
                        double longitude) {
        super(numberOfRooms, price, area);
        this.numberOfFloors = numberOfFloors;
        setLocation(latitude, height, longitude);
    }

    @Override
    public String toString() {
        return "\n\nPersonalHome:" +
                "\nnumberOfFloors = " + numberOfFloors +
                "\nlocation = " + getLocation() +
                "\nnumber of rooms = " + getNumberOfRooms() +
                "\nprice = " + getPrice() +
                "\narea = " + getArea();
    }


    public int getNumberOfFloors() {
        return numberOfFloors;
    }

    public void setNumberOfFloors(int numberOfFloors) {
        this.numberOfFloors = numberOfFloors;
    }
}

File: Lab 13 (lab 4 python)/app/personal_home.py
from app.accommodation import Accommodation


class PersonalHome(Accommodation):
    def __init__(self, number_of_rooms, price, area, latitude, number_of_floors):
        super(PersonalHome, self).__init__(number_of_rooms, price, area, latitude)
        self.number_of_floors = number_of_floors

    def __str__(self):
        return super(PersonalHome, self).__str__() + ' number of floors = ' \
               + str(self.number_of_floors)